LaCE-onboarding-guide.md

```diff
@@ -123,6 +123,9 @@ This page will help you solve common queries that you may face while working on
 
 ![Dummy-Math-Classroom](images/LaCEOnboardingGuide/Dummy-Math-Classroom.png)
 
+*Note*: If we face the error `Server error: 'None Type' object has no attribute 'version'`, go to `/release-coordinator page`(the user should have release-coordinator role), and under the MISC tab, click on `Flush Cache` button.
```

Comment:
we face --> you encounter

Maybe, since these are introductory instructions, better to provide step-by-step instructions for how to get the RC role? So, basically, something like:

```
*Note*: If you encounter the error XYZ, do the following steps to fix it:

1. Go to /admin > Roles and make sure your username is assigned the "release coordinator" role.
2. Go to /release-coordinator and ...
```